src/System.IO.Pipelines.Networking.Tls/CipherInfo.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace System.IO.Pipelines.Networking.Tls
+{
+    public struct CipherInfo
+    {
+        public string Name;
+        public int KeySizeInBits;
```

Comment:
Yes that was a hang over from testing/checking negotiated ciphers during perf testing. I will remove it